Simplified/NYPLReaderReadiumView.m

```diff
@@ -497,6 +526,15 @@ - (void)readiumInitialize
   
   dispatch_async(dispatch_get_global_queue(DISPATCH_QUEUE_PRIORITY_BACKGROUND, 0),^{
     [self calculateBookLength];
+    
```

Comment:
@winniequinn I realized there's not yet valid data in `self.bookMapDictionary` until after this asynchronous step. I just want to make sure you think it's okay to delay creating and checking the sync/book read status until this has been done, so that the alert will say "Chapter X" and not be NULL or generic.